<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KKULLANPF1GQ45C" ignored="0" name="Test1" tests="6" failures="3" timestamp="21 Jul 2019 10:29:41 GMT" time="60.599" errors="0">
  <testcase name="loginTest" time="11.592" classname="com.nopcommerce.testCases.TC_LoginTest_001"/>
  <testcase name="loginTest" time="8.496" classname="com.nopcommerce.testCases.TC_LoginDDT_002"/>
  <testcase name="loginTest" time="3.543" classname="com.nopcommerce.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopcommerce.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="3.369" classname="com.nopcommerce.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopcommerce.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="3.496" classname="com.nopcommerce.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.nopcommerce.testCases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:45)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="addNewCustomer" time="19.28" classname="com.nopcommerce.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- Test1 -->
